client/src/components/DatasetInformation/DatasetAttributes.test.js

```diff
@@ -0,0 +1,62 @@
+import Vue from "vue";
+import axios from "axios";
+import { mount } from "@vue/test-utils";
+import { getLocalVue } from "jest/helpers";
+import DatasetAttributes from "./DatasetAttributes";
+import MockProvider from "../providers/MockProvider";
+import MockAdapter from "axios-mock-adapter";
+
+const localVue = getLocalVue();
+const flushPromises = () => new Promise((resolve) => setTimeout(resolve));
```

Comment:
```suggestion
import MockAdapter from "axios-mock-adapter";
import flushPromises from "flush-promises";

const localVue = getLocalVue();
```

Reply:
It'd be best to use the same flushPromises we use elsewhere in testing.  It's (obviously) pretty much the same under the hood, but consistency is good.